packages/tools/src/stateManagement/annotation/FrameOfReferenceSpecificAnnotationManager.ts

```diff
@@ -179,6 +180,7 @@ export default class FrameOfReferenceSpecificAnnotationManager {
 
     toolSpecificAnnotations.push(annotation);
     checkAndDefineIsLockedProperty(annotation);
+    checkAndDefineIsVisibleProperty(annotation);
```

Comment:
This (and also locked property line above) causes a code smell. I.e there is an inappropriate intimacy between AnnotationManger and Each Tool, but they should be actually totally decoupled.

My recommendation is to:
base/AnnotationTool be responsible decorate the annotation (on addnew annotation method)

Each Tool to invoke its super method to allow this decoration.

Reply:
ok
https://github.com/cornerstonejs/cornerstone3D-beta/issues/95